.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ aliases:
 executors:
   node:
     docker:
-      - image: circleci/node:8-browsers
+      - image: circleci/node:10.16-stretch-browsers
```

Comment:
It sounds dangerous to not run tests on Node 8 anymore 🤔

Reply:
Undid this change and turned off the v8 Compile Cache `ncc` feature.